node/src/runtime/mainnet.rs

```diff
@@ -150,46 +150,6 @@ pub fn testnet_genesis(
 					.into_iter()
 					.collect();
 
-				map.insert(
```

Comment:
The issue with this code it bumps issuance. Basically with this code we have u128::MAX issuance. 
I don't think we actually need this code.